packages/components/carousel/src/carousel.vue

```diff
@@ -55,7 +57,7 @@
         @mouseenter="throttledIndicatorHover(index)"
         @click.stop="handleIndicatorClick(index)"
       >
-        <button :class="ns.e('button')">
+        <button :class="ns.e('button')" aria-label="Carousel Button">
```

Comment:
It may be clearer to change the label content to `carousel indicator`.

Reply:
Done